src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -123,6 +145,24 @@
     )
     private File outputFile;
 
+    @Argument(
+            doc = "Intervals are layout in a grid so that bin start at fixed positions (grid points) 1 + n * bin_length. When intervals " +
```

Comment:
If true, bins are laid out in a grid in which each bin start occurs at 1 + n * bin_length.  When the specified intervals do not exactly align with the grid (after optional padding), the first and/or last bin in that interval will be smaller than bin-length and may be dropped (see minimum-bin-length).  If false, bins starts will coincide with the starts of the specified (and optionally padded) intervals.